docs/2.deploy/20.providers/genezio.md

```diff
@@ -0,0 +1,56 @@
+# Genezio
+
+> Deploy Nitro apps to Genezio.
+
+**Preset:** `genezio`
+
+:read-more{title="Genezio" to="https://genezio.com"}
+
+# Project Setup
+
+## 1. Install genezio CLI globally
+
+```bash
+npm install genezio -g
+```
+
+## Build and deploy
+#### Build your Nitro project using the `Genezio` preset:
+```bash
+nitro build --preset genezio
+```
+Building the project will automatically create a default `genezio.yaml` configuration file, if it doesn't already exist.
+
+To further customize the file to your needs, you can consult the
+[official documentation](https://genezio.com/docs/project-structure/genezio-configuration-file/).
+
+## 3. Deploy your project
+
+Run the following command in your terminal:
+
+```bash
+genezio deploy
+```
+
+If your application use environment variables, you can deploy them using the following command:
+
+```bash
+genezio deploy --env <path-to-your-env-file>
+```
+
+:::info
+You need to deploy your environment variables single time.
+After that, you can deploy your project without the `--env` flag.
+:::
```

Comment:
This is subject to change in the future - there is a task to improve setting environment variables experience.

I would transform this into something like:
```
For projects that set environment backends, you can check the following tutorial: [Genezio - Environment Variables].
```